infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/devfile/DockerimageComponentToWorkspaceApplier.java

```diff
@@ -9,16 +9,16 @@
  * Contributors:
  *   Red Hat, Inc. - initial API and implementation
  */
-package org.eclipse.che.api.devfile.server.convert.component.dockerimage;
+package org.eclipse.che.workspace.infrastructure.kubernetes.devfile;
```

Comment:
From the first look, it's not clear why Dockerimage component related classes are in k8s module.
Maybe it makes sense to add into java docs that this class converts dockerimage component to k8s infra understood format?